tests/test_system_filesystemspace.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2019 FreeIPA Contributors see COPYING for license
+#
+
+from base import BaseTest
+from unittest.mock import Mock
+from util import capture_results
+from collections import namedtuple
+
+from ipahealthcheck.core import config, constants
+from ipahealthcheck.system.plugin import registry
+from ipahealthcheck.system.filesystemspace import FileSystemSpaceCheck
+
+
```

Comment:
Can you add a success test case as well?

Reply:
Yes, doin' so

Reply:
Done. There are three tests now.